docs/collector.thermalzone.md

```diff
@@ -0,0 +1,31 @@
+# tcp collector
+
+The thermalzone collector exposes metrics about system temps. Note that temperature is given in Kelvin
+
+|||
+-|-
+Metric name prefix  | `thermalzone`
+Classes             | [`Win32_PerfRawData_Counters_ThermalZoneInformation`](https://msdn.microsoft.com/en-us/library/aa394341(v=vs.85).aspx)
```

Comment:
The link seems off? I get TCP/IP docs.

Reply:
Woops, I took the tcp collector .md as a template for this. Clearly missed some stuff lol. fixed.